tests/test_svg_image.py

```diff
@@ -417,6 +417,32 @@ def test_view_box_re(self):
             (".1,0,0,0", ViewBox(0.1, 0, 0, 0)),
             ("+.1,0,0,0", ViewBox(0.1, 0, 0, 0)),
             ("-.1,0,0,0", ViewBox(-0.1, 0, 0, 0)),
+            # https://github.com/wagtail/Willow/issues/127
+            (
+                "-5.000000011410315e-06 0.0 750.00001 525.000007",
+                ViewBox(-5.000000011410315e-06, 0.0, 750.00001, 525.000007),
```

Comment:
can you add a test for `-5.000000011410315e06` which should be valid as far as I can tell from https://www.w3.org/TR/SVG11/types.html#DataTypeNumber and https://www.w3.org/TR/SVG11/types.html#DataTypeInteger ?

Other than that, LGTM

Reply:
Good point, added a few more tests.